bftengine/src/bftengine/CollectorOfThresholdSignatures.hpp

```diff
@@ -396,7 +396,7 @@ class CollectorOfThresholdSignatures {
           // if verification failed again
           // signer index starts with 1, therefore shareId-1
```

Comment:
if this is not true anymore remove the comment

Reply:
done